torch/csrc/jit/pickler.cpp

```diff
@@ -446,10 +461,38 @@ OpCode Unpickler::readInstruction() {
       break;
     case OpCode::GLOBAL: {
       // Module name, it's not needed for anything
-      readString();
+      auto module_name = readString();
+      if (module_name == "__main__") {
```

Comment:
weird blank if statement. Better to assert if it always is true or omit otherwise;

```
auto module_name = readString();
(void) module_name; // silence unused warning.
```